pkg/docker/config/config.go

```diff
@@ -37,6 +38,7 @@ var (
 	xdgRuntimeDirPath       = filepath.FromSlash("containers/auth.json")
 	dockerHomePath          = filepath.FromSlash(".docker/config.json")
 	dockerLegacyHomePath    = ".dockercfg"
+	nonLinuxAuthFilePath    = filepath.FromSlash(".containers/auth.json")
```

Comment:
We use `~/.config/containers` for `policy.json`, `registries.conf*`, `certs.d`, so it would make sense to me to use the same directory here as well.